### PR TITLE
bugfix/16750-mapbubble-joinBy

### PR DESCRIPTION
**Pull request by madepiet**
Default joinBy added for mapbubble.

### CODE REVIEW

**Comment by TorsteinHonsi** [comment]
Regarding the Safari fails: I have fixed the first one in another PR. The second one I don't recognize.

File: samples/unit-tests/maps/mapbubble-linewidth/demo.js [hunk 1]
```diff
@@ -51,4 +51,10 @@ QUnit.test('MapBubble', function (assert) {
         `When hovering over mapbubble series with shared tooltip,
         there should be no errors in the console.`
     );
+
+    // Passing the default joinBy value for MapBubbleSeries (#16750).
+    assert.ok(
```
Comment (raf18seb) [inline samples/unit-tests/maps/mapbubble-linewidth/demo.js:56]: Could you use `assert.strictEqual` for consistency?

File: samples/unit-tests/maps/mapbubble-linewidth/demo.js [hunk 2]
```diff
@@ -51,4 +51,10 @@ QUnit.test('MapBubble', function (assert) {
...
+        chart.series[1].joinBy[0] === "hc-key",
```
Comment (raf18seb) [inline samples/unit-tests/maps/mapbubble-linewidth/demo.js:57]: Please use single quotes when writing a JavaScript code

File: samples/unit-tests/maps/mapbubble-linewidth/demo.js [hunk 3]
```diff
@@ -51,4 +51,10 @@ QUnit.test('MapBubble', function (assert) {
...
+        `Checking default options for joinBy: 'hc-key'.`
```
Comment (raf18seb) [inline samples/unit-tests/maps/mapbubble-linewidth/demo.js:58]: One of our internal rules, when writing unit tests, is that description of a test should have a "should" keyword.

For example:
```
The value of joinBy in the mapbubble series should be 'hc-key' by default.
```

**Review by raf18seb — CHANGES_REQUESTED** [review]

**Review by raf18seb — APPROVED** [review]
Thanks!
(Now only failed tests left to resolve) - it's actually done, so all good!

**Review by DJTechnoo — APPROVED** [review]

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**